src/core/utils/bin-sorter.js

```diff
@@ -79,7 +79,9 @@ export default class BinSorter {
    * @return {Number | Boolean} max count
    */
   getMaxCount() {
-    return Math.max(...this.sortedBins.map(b => b.counts));
+    let maxCount = 0;
```

Comment:
Nit: Just call it `max` or `maxValue`? It does not seem like a "count"...

Reply:
trying to match with the function name.